angular/projects/admin-nrpti/src/app/records/utils/record-utils.ts

```diff
@@ -230,4 +230,46 @@ export class RecordUtils {
         return null;
     }
   }
+
+  async handleDocumentChanges(links = [], documents = [], documentsToDelete = [], recordId, factoryService) {
```

Comment:
Id recommend adding some types to these arrays. Ex: `links = string[]` etc

I would also be my preference to throw some JSDoc on this function, since its not particularly clear what these arrays hold.  Ex: Does `documentsToDelete` have whole document objects in it? or Just document IDs?